en/04_Changelogs/5.0.0.md

```diff
@@ -940,182 +940,182 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Removed deprecated method `SilverStripe\Control\HTTPRequest::detect_method()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Control\SimpleResourceURLGenerator::resolveUnsecuredResource()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Core\BaseKernel::getIgnoredCIConfigs()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Core\BaseKernel::sessionEnvironment()` without equivalent functionality to replace it
-- Removed deprecated method `SilverStripe\Core\ClassInfo::baseDataClass()` - use `DataObject::getSchema()->baseDataClass()`
-- Removed deprecated method `SilverStripe\Core\ClassInfo::table_for_object_field()` - use `DataObject::getSchema()->tableForField()`
-- Removed deprecated method `SilverStripe\Core\Config\Config_ForClass::update()` - use `merge()` instead
+- Removed deprecated method `SilverStripe\Core\ClassInfo::baseDataClass()` - use [`SilverStripe\ORM\DataObjectSchema::baseDataClass()`](api:SilverStripe\ORM\DataObjectSchema::baseDataClass())
+- Removed deprecated method `SilverStripe\Core\ClassInfo::table_for_object_field()` - use [`SilverStripe\ORM\DataObjectSchema::tableForField()`](api:SilverStripe\ORM\DataObjectSchema::tableForField())
+- Removed deprecated method `SilverStripe\Core\Config\Config_ForClass::update()` - use [`SilverStripe\Core\Config\Config_ForClass::merge()`](api:SilverStripe\Core\Config\Config_ForClass::merge()) instead
 - Removed deprecated method `SilverStripe\Core\Config\Configurable::set_stat()` - use `config()->set()` instead
 - Removed deprecated method `SilverStripe\Core\Config\Configurable::stat()` - use `config()->get()` instead
-- Removed deprecated method `SilverStripe\Core\Convert::array2json()` - use `json_encode()` instead
-- Removed deprecated method `SilverStripe\Core\Convert::json2array()` - use `json_decode($val, true)` instead
-- Removed deprecated method `SilverStripe\Core\Convert::json2obj()` - use `json_decode()` instead
-- Removed deprecated method `SilverStripe\Core\Convert::raw2json()` - use `json_encode()` instead
+- Removed deprecated method `SilverStripe\Core\Convert::array2json()` - use [`SilverStripe\Core\Convert::json_encode()`](api:SilverStripe\Core\Convert::json_encode()) instead
+- Removed deprecated method `SilverStripe\Core\Convert::json2array()` - use [`SilverStripe\Core\Convert::json_decode()`](api:SilverStripe\Core\Convert::json_decode()) instead
+- Removed deprecated method `SilverStripe\Core\Convert::json2obj()` - use [`SilverStripe\Core\Convert::json_decode()`](api:SilverStripe\Core\Convert::json_decode()) instead
+- Removed deprecated method `SilverStripe\Core\Convert::raw2json()` - use [`SilverStripe\Core\Convert::json_encode()`](api:SilverStripe\Core\Convert::json_encode()) instead
```

Comment:
```suggestion
- Removed deprecated method `SilverStripe\Core\Convert::array2json()` - use [`json_encode()`](https://www.php.net/manual/en/function.json-encode.php) instead
- Removed deprecated method `SilverStripe\Core\Convert::json2array()` - use [`json_decode()`](https://www.php.net/manual/en/function.json-decode.php) instead
- Removed deprecated method `SilverStripe\Core\Convert::json2obj()` - use [`json_decode()`](https://www.php.net/manual/en/function.json-decode.php) instead
- Removed deprecated method `SilverStripe\Core\Convert::raw2json()` - use [`json_encode()`](https://www.php.net/manual/en/function.json-encode.php) instead
```
These are built-in functions, not part of the `Convert` class

Reply:
Done